### PR TITLE
Fix cleanup_tasks_log stepname and make period configurable

### PR DESCRIPTION
**Pull request by Mark90**
The core workflow `cleanup_tasks_log` currently logs `"Clean up tasks older than one week"` while it is actually cleaning tasks older than 3 days. 

This PR fixes changes that to `"Clean up completed tasks older than TASK_LOG_RETENTION_DAYS"` while making the value configurable through `TASK_LOG_RETENTION_DAYS`

### CODE REVIEW

File: orchestrator/settings.py [hunk 1]
```diff
@@ -77,6 +77,7 @@ class AppSettings(BaseSettings):
     DEFAULT_CUSTOMER_FULLNAME: str = "Default::Orchestrator-Core Customer"
     DEFAULT_CUSTOMER_SHORTCODE: str = "default-cust"
     DEFAULT_CUSTOMER_IDENTIFIER: str = "59289a57-70fb-4ff5-9c93-10fe67b12434"
+    DAYS_TO_KEEP_OLD_TASKS: int = 3
```
Comment (dmarjenburgh) [inline orchestrator/settings.py:80]: Ik vind iets als `TASK_LOG_RETENTION_DAYS` oid mooier voor een config naam

File: orchestrator/workflows/tasks/cleanup_tasks_log.py [hunk 2]
```diff
@@ -15,21 +15,20 @@
...
 from orchestrator.db import ProcessTable, db
+from orchestrator.settings import app_settings
 from orchestrator.targets import Target
 from orchestrator.types import State
 from orchestrator.utils.datetime import nowtz
 from orchestrator.workflow import ProcessStatus, StepList, done, init, step, workflow
 
 
-@step("Clean up tasks older than one week")
+@step(f"Clean up completed tasks older than {app_settings.DAYS_TO_KEEP_OLD_TASKS} days")
```
Comment (dmarjenburgh) [inline orchestrator/workflows/tasks/cleanup_tasks_log.py:25]: Dynamische step names kunnen wel gevaarlijk zijn

File: test/unit_tests/graphql/test_product_blocks.py [hunk 3]
```diff
@@ -107,6 +107,7 @@ def test_product_blocks_query(test_client):
             "inUseBy": [],
         },
     ]
+    product_blocks.sort(key=lambda x: x["name"])  # No sort in the query; sort before the assert to prevent flaky tests
```
Comment (dmarjenburgh) [inline test/unit_tests/graphql/test_product_blocks.py:110]: Mooier om sets te gebruiken bij de assert

File: orchestrator/settings.py (hunk 1 above)
Comment (Mark90) [reply orchestrator/settings.py]: Good suggestion, changed it.

File: orchestrator/workflows/tasks/cleanup_tasks_log.py (hunk 2 above)
Comment (Mark90) [reply orchestrator/workflows/tasks/cleanup_tasks_log.py]: Yeah I was on the fence about this. Changed it to just print the name of the env-var.

File: test/unit_tests/graphql/test_product_blocks.py (hunk 3 above)
Comment (Mark90) [reply test/unit_tests/graphql/test_product_blocks.py:110]: Not possible to hash dicts

Comment (mrijk) [reply test/unit_tests/graphql/test_product_blocks.py:110]: you can also use` key=itemgetter("name")`

**Pull request merged**